developers/prerequisites/OSGi-tasks.md

```diff
@@ -0,0 +1,198 @@
+---
+layout: developersguide
+---
```

Comment:
Could you please add a title attribute?
This feature is quite new: https://github.com/openhab/openhab-docs/pull/58